lib/commons/aria/index.js

```diff
@@ -1407,9 +1407,11 @@ lookupTable.role = {
 					'h5',
 					'h6',
 					'header',
+					'iframe',
 					'li',
+					'ol',
 					'section',
-					'ol'
+					'ul'
```

Comment:
Why this addition?

Reply:
Not sure how this got overlooked, but it the spec says these are allowed.